src/onnx/mod.rs

```diff
@@ -102,9 +102,16 @@ impl Environment {
             .map_err(|_| NewSessionError::MalformedModelPath)?;
         unsafe {
             let allocator = self.api.get_allocator_with_default_options()?;
+            let sess_options = scopeguard::guard(self.api.create_session_options()?, |ptr| {
+                self.api.release_session_options(ptr)
+            });
+
+            #[cfg(feature = "cuda")]
```

Comment:
Gotta add `cuda = []` to Cargo.toml too under features.

Reply:
oh yeah I added that but accidentally forgot to commit it before force pushing